usage.txt

```diff
@@ -26,6 +26,7 @@ out                the dir to put the app into at the end. defaults to current w
 overwrite          if output directory for a platform already exists, replaces it rather than skipping it
 prune              runs `npm prune --production` on the app
 sign               should contain the identity to be used when running `codesign` (only for building for the darwin platform, on OS X)
+strict-ssl         whether SSL certificates are required to be valid when downloading Electron (defaults to true)
```

Comment:
I probably should have caught this first, but how are you supposed to call this on the command line? I think this is the first boolean flag in `electron-packager` that defaults to `true`.